lib/mongoid/history/trackable.rb

```diff
@@ -282,6 +282,8 @@ def track_history_for_action(action)
 
           clear_trackable_memoization
 
+          return unless block
```

Comment:
The ruby way is to do `unless block_given?` and to mark the block above as `_block`.

Reply:
Good feedback, thanks!

Reply:
Turns out, with `block_given?` there's no need for a `&block` param!